src/app/components/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -7,19 +7,21 @@
 <div class="f8-wi__list-entry">
 	<!-- info area -->
   <div class="list-view-pf-main-info">
-    <div class="list-view-pf-left type f8-wi__list-witype">
-      <span class="pull-left">
+    <div class="list-view-pf-left type pull-left f8-wi__list-witype">
+      <span class="pull-left margin-0">
         {{workItem.attributes['system.number']}}
       </span>
-      <span class="color-grey pull-left fa
+      <span class="color-grey pull-left margin-h-10
```

Comment:
What does `margin-h-10` refer too? Is this controlled in some variable file, or is it a custom class?

Reply:
margin-h-10 refers to margin left and right 10px. Its in formulas.less